system/Validation/Exceptions/ValidationException.php

```diff
@@ -15,27 +15,27 @@
 
 class ValidationException extends FrameworkException
 {
-    public static function forRuleNotFound(?string $rule = null)
+    public static function forRuleNotFound(?string $rule = null): ValidationException
```

Comment:
This is a breaking change.
Please use doc comment.

Reply:
Done.